app/components/carbon-calculator.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="footprint rounded-3">
-  <p class="note">Note: The limit of carbon foot print usable is 60 tons</p>  
+  <p class="note">Note: The limit of carbon foot print usable is 60 tons</p>
   <p class="desc p-2">Make sure your carbon-footprint usage is less than the
```

Comment:
change it as "Make sure to optimize the energy usage to cope up with your threshold limit

Reply:
changed